translations/messages.nl.yml

```diff
@@ -12,8 +12,8 @@ page.navigation.route.sp_demo: Demo SP
 
 page.footer.terms: Terms and Conditions
 
-page.index.title: Welcome to the Azure MFA GSSP
-page.index.body.intro: TODO Introduction
+page.index.title: Welcome to the Azure MFA
+page.index.body.intro: This is the landingpage for the Azure MFA plugin in OpenConext-Stepup.
```

Comment:
See comment in the English translation file. 

Some of the Dutch translations seem to be missing here, something you might want to address before a first release?